test-data/unit/check-protocols.test

```diff
@@ -1663,8 +1663,8 @@ def apply_gen(f: Callable[[T], T]) -> T:
 reveal_type(apply_gen(Add5())) # E: Revealed type is 'builtins.int*'
 def apply_str(f: Callable[[str], int], x: str) -> int:
     return f(x)
-apply_str(Add5(), 'a') # E: Argument 1 to "apply_str" has incompatible type "Add5"; expected Callable[[str], int] \
-                       # N: 'Add5.__call__' has type 'Callable[[Arg(int, 'x')], int]'
+apply_str(Add5(), 'a') # E: Argument 1 to "apply_str" has incompatible type "Add5"; expected "Callable[[str], int]" \
+                       # N: 'Add5.__call__' has type "Callable[[Arg(int, 'x')], int]"
```

Comment:
Another potential idea: maybe use double quotes instead of single quotes in ``'Add5.__call__'``? (But not in this PR.)